io/src/main/java/org/commonjava/maven/ext/io/rest/DefaultTranslator.java

```diff
@@ -58,14 +58,15 @@
     {
     };
 
-    public enum ENDPOINT {
+    public enum Endpoint
+    {
         LOOKUP_GAVS ("lookup/maven"),
         LOOkUP_LATEST ("lookup/maven/latest");
 
         @Getter
         private final String endpoint;
 
-        ENDPOINT( String s )
```

Comment:
Should we override `toString()` to return `getEndpoint()`? I sort of expected this so that I could build a URL using `"/" + LOOKUP_GAVS`. What do you think?

Reply:
Thats a good idea. Could probably remove the `@Getter` maybe?